kvmagent/tests/test_cpu_hotplug_online.py

```
@@ -0,0 +1,144 @@
+                    'online': True
+                })
+
+        self.assertEqual(len(offline_cpus), 0, "No offline CPUs should be found")
+
+    def test_qga_exception_is_swallowed(self):
+        """QGA failure must not affect hotplug - exception should be caught."""
+        caught = False
+        try:
+            # Simulate the try/except pattern from _qga_online_hotplugged_cpus
+            raise Exception("QGA connection timeout")
+        except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)